src/main/scala/com.snowplowanalytics.snowplow.micro/SqliteStorage.scala

```diff
@@ -38,15 +38,19 @@ private[micro] class SqliteStorage(xa: Transactor[IO]) extends EventStorage {
       val eventData = events.zip(eventJsons).map { case (goodEvent, eventJson) =>
         val appId = goodEvent.event.app_id
         val eventName = goodEvent.event.event_name
-        (goodEvent.event.event_id.toString, goodEvent.event.collector_tstamp, eventJson.noSpaces, goodEvent.incomplete, appId, eventName)
+        val platform = goodEvent.event.platform
+        val nameTracker = goodEvent.event.name_tracker
+        val domainUserid = goodEvent.event.domain_userid
+        val vTracker = goodEvent.event.v_tracker
```

Comment:
I fee like this one is not as useful as the others

Reply:
Was matching we do in console for collection volumes - happy to drop it out though if you don't like it.